diff --git a/components/sync/protocol/proto_visitors.h b/components/sync/protocol/proto_visitors.h
index 55784eba3d383888f8db7a22f9d4034b9db82a78..1fcca874d0034977ebb12b401576801093db486b 100644
--- a/components/sync/protocol/proto_visitors.h
+++ b/components/sync/protocol/proto_visitors.h
@@ -400,6 +400,7 @@ VISIT_PROTO_FIELDS(const sync_pb::DeviceInfoSpecifics& proto) {
   VISIT(manufacturer);
   VISIT(last_updated_timestamp);
   VISIT(feature_fields);
+  BRAVE_VISIT_DEVICE_INFO_SPECIFICS_BRAVE_FIELDS
   VISIT(sharing_fields);
   VISIT(invalidation_fields);
   VISIT(paask_fields);
@@ -500,6 +501,7 @@ VISIT_PROTO_FIELDS(const sync_pb::EntitySpecifics& proto) {
   VISIT(typed_url);
   VISIT(user_consent);
   VISIT(user_event);
+  BRAVE_VISIT_ENTITY_SPECIFICS_VG_BODY_VG_SPEND_STATUS
   VISIT(wallet_metadata);
   VISIT(web_app);
   VISIT(wifi_configuration);
@@ -1330,7 +1332,7 @@ VISIT_PROTO_FIELDS(
   VISIT(width);
   VISIT(height);
 }
-
+BRAVE_VISIT_PROTO_FIELDS_BRAVE_SPECIFIC_FIELDS_VG_BODY_SPECIFICS_VG_SPEND_STATUS_SPECIFICS
 }  // namespace syncer
 
 #undef VISIT_
